src/adtProjections.F90

```diff
@@ -1287,7 +1287,7 @@ subroutine searchQuads(pts, conn, searchPts, nPts, nConn, nSearchPts, faceID, uv
     do i=1, nSearchPts
        coor(1:3) = searchPts(:, i)
        coor(4) = 1e20
-       call minDistanceTreeSearchSinglePoint(ADT, coor, intInfo, uvw, dummy, &
+       call minDistTreeSearchSinglePoint(ADT, coor, intInfo, uvw, dummy, &
```

Comment:
I almost feel that this is a step backwards. Since pyspline does not require tapenade I would be more in favor of using 2003/2008.

Reply:
Let me update to F2008 then, and maybe update the intrinsics at the same time.